tests/fixtures/user-batchupdate-event-payload.json

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
the payloads used for batch updates are actually extract files from the platform. To test this scenario, I would use data pulled from actual extracts and have this passed through the extract handler since we want to run the test against that part too to make sure changes in the extractHandler logic does not introduce any regressions.

Reply:
that makes sense, I thought the same thing myself.  but we're going to have to mock the s3 call to get the message right?  I didn't know how far we wanted to go in testing the extractHandler specifically, I thought having it test the logic was enough for this particular fix.

Reply:
yup, can do